pkg/services/ngalert/metrics/multi_org_alertmanager.go

```diff
@@ -119,12 +189,35 @@ func NewAlertmanagerAggregatedMetrics(registries *metrics.TenantRegistries) *Ale
 			fmt.Sprintf("%s_%s_silences", Namespace, Subsystem),
 			"How many silences by state.",
 			[]string{"org", "state"}, nil),
+
+		dispatchAggrGroups: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_dispatcher_aggregation_groups", Namespace, Subsystem),
+			"Number of active aggregation groups",
+			nil, nil),
+		dispatchProcessingDuration: prometheus.NewDesc(
+			fmt.Sprintf("%s_%s_dispatcher_alert_processing_duration_seconds", Namespace, Subsystem),
+			"Summary of latencies for the processing of alerts.",
+			nil, nil),
```

Comment:
For reviews: [Upstream dispatcher metrics](https://github.com/prometheus/alertmanager/blob/main/dispatch/dispatch.go#L42)